ui/src/app/components/runtime-configuration-panel/customize-panel.spec.tsx

```diff
@@ -383,4 +385,91 @@ describe(CustomizePanel.name, () => {
     expect(screen.queryByText(/Warning Number One/)).toBeInTheDocument();
     expect(screen.queryByText(/Warning Number Two/)).toBeInTheDocument();
   });
+
+  // many subcomponents are enabled/disabled by the same logic.  PresetSelector can act as a stand-in here for:
+  // MachineSelector, GpuConfigSelector, DataProcConfigSelector, the autopause Dropdown, DiskSelector, and
+  // Delete Environment in CustomizePanelFooter.
+  // The runtime-compute Dropdown has similar logic, but only when allowDataproc is also true
+
+  it('enables the PresetSelector when no runtimeExists, regardless of runtimeStatus', async () => {
```

Comment:
when no runtimeExists, that means we're configuring a runtime that we want to create